### PR TITLE
added the health/liveness endpoint for kubernetes

### PR DESCRIPTION
**Pull request by KevinJCross**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-908

### Change description ###
- added the custom health additions of liveness (does nothing
- moved the health from /actuator/health to /health
- added tests.

### CODE REVIEW

File: src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java [hunk 1]
```diff
@@ -69,4 +72,27 @@ public void should_be_able_to_delete_all_users_drafts_in_service() throws Except
         draftStoreClient.deleteAllUsersDrafts();
         assertThat(draftStoreClient.readDraftPage().data).isEmpty();
     }
+
```
Comment (timja) [inline src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java]: unneeded blank line?

File: src/main/java/uk/gov/hmcts/reform/draftstore/service/Liveness.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.draftstore.service;
+
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.stereotype.Component;
+
```
Comment (timja) [inline src/main/java/uk/gov/hmcts/reform/draftstore/service/Liveness.java]: this blank line needed?

File: src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java [hunk 3]
```diff
@@ -90,6 +93,6 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
             .registerModule(new JavaTimeModule())
             .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-            .setDateFormat(new ISO8601DateFormat());
+            .setDateFormat(new StdDateFormat());
```
Comment (timja) [inline src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java]: what's the difference? was the old one deprecated?

Comment (KevinJCross) [reply src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java]: yup the its deprecated to be replaced with StdDateFormat.

Comment (KevinJCross) [reply src/main/java/uk/gov/hmcts/reform/draftstore/config/DraftStoreConfig.java]: hmmm ... my appetite for risk has suddenly disappeared and Ill revert this.

**Comment by timja** [comment]
This was merged separately

**Pull request closed without merging**